edx_event_bus_kafka/internal/tests/test_producer.py

```diff
@@ -137,3 +139,39 @@ def test_send_to_event_bus(self, mock_get_serializers):
             on_delivery=ep.on_event_deliver,
             headers={'ce_type': 'org.openedx.learning.auth.session.login.completed.v1'},
         )
+
+    def test_polling_loop(self):
+        with override_settings(
+                EVENT_BUS_KAFKA_POLL_INTERVAL_SEC=0.1
+        ):
+            call_count = 0
+
+            def increment_call_count(*args):
+                nonlocal call_count
+                call_count += 1
+                if call_count == 1:
+                    raise Exception("Exercise error handler on first iteration")
+
+            mock_producer = Mock(**{'poll.side_effect': increment_call_count})
+
+            producer_api = ep.EventProducerKafka(mock_producer)  # Created, starts polling
+
+            # Allow a little time to pass and check that the mock poll has been called
+            time.sleep(1.0)
+            assert call_count > 0
+            print(producer_api)  # Use the value here to ensure it isn't GC'd early
```

Comment:
I'm surprised pylint didn't complain about the 'print' statement. Is that just because we haven't 100% set it up?

Reply:
Yeah, I'm a little surprised. But maybe we have that linter rule disabled/not enabled? Pylint is definitely running on this file, though.